code/game/objects/items/tools/mining_tools.dm

```diff
@@ -98,6 +86,261 @@
 	drill_verb = "drilling"
 
 
+/obj/item/tool/pickaxe/plasmacutter
+	name = "plasma cutter"
+	icon_state = "plasma_cutter_on"
+	var/base_icon_state = "plasma_cutter_on"
+	item_state = "gun"
+	w_class = 3.0 //it is smaller than the pickaxe
```

Comment:
Due to be a big game-changing tool it should be much bigger. Make it fit in the back slot, not inside a backpack.